src/http.rs

```diff
@@ -125,14 +125,14 @@ impl NgrokHttpTunnelBuilder {
     /// OIDC configuration.
     /// If not called, OIDC is disabled.
     ///
-    /// :param str provider: The name of the OIDC provider to use.
+    /// :param str issuer_url: The name of the OIDC issuer_url to use.
```

Comment:
`The name of the OIDC issuer URL to use`?

Reply:
Yes, updated!